microservice/microservice/Targets/Beamable.Microservice.Runtime.props

```diff
@@ -28,6 +28,19 @@
         <GenerateClientCode Condition="'$(GenerateClientCode)' == ''">true</GenerateClientCode>
     </PropertyGroup>
 
+    <!-- Set some default Beamable options, https://docs.beamable.com/docs/cli-guide-microservice-configuration -->
+    <PropertyGroup>
+        <!-- By default, if the BeamId hasn't been set, we use the name of the file -->
+        <BeamId Condition="'$(BeamId)'==''">$(MSBuildProjectName)</BeamId>
+    </PropertyGroup>
+    
+    <PropertyGroup Label="Resolve .beamable paths">
+        <BeamableProjectRoot Condition="'$(BeamableProjectRoot)'==''">$([MSBuild]::GetDirectoryNameOfFileAbove("$(MSBuildProjectDirectory)/..", ".beamable/connection-configuration.json"))</BeamableProjectRoot>
+        <BeamableFolderPath Condition="'$(BeamableFolderPath)'==''">$(BeamableProjectRoot)/.beamable</BeamableFolderPath>
+        <BeamableLocalDevProjectPath Condition="'$(BeamableLocalDevProjectPath)'==''">$(BeamableFolderPath)/temp/localDev</BeamableLocalDevProjectPath>
+        <BeamableLocalDevProjectsGlob Condition="'$(BeamableLocalDevProjectsGlob)'==''">$(BeamableLocalDevProjectPath)/*.csproj</BeamableLocalDevProjectsGlob>
```

Comment:
I'm a bit lost with this, what are these csproj files supposed to be? Why are they in a temp folder?